.github/workflows/ci.yml

```diff
@@ -44,7 +44,7 @@ jobs:
           channel-priority: true
           environment-file: devtools/conda-envs/env.yaml
           use-mamba: true
-          miniforge-variant: Mambaforge
+          miniforge-variant: Miniforge3
```

Comment:
https://github.com/conda-incubator/setup-miniconda?tab=readme-ov-file#example-10-miniforge

Looks like docs say you should just set miniforge-version and it'll work.

Reply:
(note this is what we do elsewhere, i.e. in MDA)